examples/01_getting_started.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+from haystack.document_stores import InMemoryDocumentStore
+from haystack.nodes import AnswerParser, PromptNode, PromptTemplate, BM25Retriever
+from haystack.pipelines import Pipeline
+from haystack.pipelines.standard_pipelines import TextIndexingPipeline
+from haystack.utils import fetch_archive_from_http
+from haystack.utils import print_answers
+
+
+def getting_started():
+    # This getting_started shows you how to use LLMs with your data, a technique called Retrieval Augmented Generation - RAG
+
+    # We are model agnostic :) Here, you can choose from: "Anthropic", "Cohere", "Huggingface", "OpenAI".
+    provider = "OpenAI"
+    # Please add an API key matching the provider.
+    API_KEY = "ADD KEY HERE"
+
+    # Loads good default pipelines for each provider.
+    indexing_pipeline, query_pipeline = load_pipelines(provider=provider, API_KEY=API_KEY)
+
+    # Downloads and adds a few Game of Thrones wiki articles to haystacks database.
+    # You can also provide a folder with your local documents. Be aware that some of your data will be sent to external APIs!
+    add_example_data(indexing_pipeline, dir="data/getting_started")
+
+    result = query_pipeline.run(query="Who is the father of Arya Stark?")
+
+    # This prints only the answer.
+    # For details, like which documents were used to generate the answer, look into the <result> object
+    print_answers(result, details="medium")
+
+
+def load_pipelines(provider, API_KEY):
+    # Query pipeline
+    document_store = InMemoryDocumentStore(use_bm25=True)
+
+    retriever = BM25Retriever(document_store=document_store, top_k=3)
+
+    question_answering_with_references = PromptTemplate(
+        prompt="deepset/question-answering-with-references",
```

Comment:
ooh just fyi, not sure if you tried it with all of the models but e.g. question answering with references performs terribly with falcon 40b (at least when I tried it). I'd suggest a safer prompt like simple question answering for the getting started just to avoid any confusion